src/SmartComponents/Recs/helpers.js

```diff
@@ -72,3 +72,27 @@ export const bulkHostActions = async ({
     });
   }
 };
+
+export const edgeSystemsCheck = async (ruleId, setSystemsCount) => {
```

Comment:
How about converting this function to return the number of conventional and edge systems count? With this would be able use those numbers in HybridInventoryTab component also to switch to edge tab when there is no conventional system, but there is edge automatically.

Reply:
I can add it in, but currently it doesnt accept that prop. We'd have to make an PR to update HybridInventory to accept it

Reply:
[hasConventionalSystems](https://github.com/RedHatInsights/insights-inventory-frontend/blob/e3e7158e47efbad4a3ab2df9b7d133c2fcd01e8e/src/components/InventoryTabs/HybridInventoryTabs.js#L23) props are designed for this. But yeah now I think that prop might have been more meaningful. It is now a bit confusing.